docs/client/components/shared/GithubButton/index.js

```diff
@@ -19,19 +19,17 @@ export default class GithubStarButton extends Component {
   }
 
   render() {
-    const size = this.props.size ? this.props.size : 'default'; // 'mega' is the other option
     const text = this.props.text ? this.props.text : 'Github';
+    const { user, repo, size } = this.props;
 
     // Note: all of the attributes including the className 'github-button' are required
     return (
       <a
         ref={(element) => { this.githubButton = element; }}
         className="github-button"
-        href="https://github.com/draft-js-plugins/draft-js-plugins"
-        data-style={size}
-        data-count-href="/draft-js-plugins/draft-js-plugins/stargazers"
-        data-count-api="/repos/draft-js-plugins/draft-js-plugins#stargazers_count"
-        data-count-aria-label="# stargazers on GitHub"
```

Comment:
hold on what's going on here? :)

Reply:
I was getting these deprecation warnings in console:

<img width="1440" alt="screen shot 2017-09-29 at 11 32 32" src="https://user-images.githubusercontent.com/1851359/31010109-ffb056e8-a509-11e7-94f2-79a94e7692b7.png">

Thus I've updated our code to use the new api according to the [github-buttons docs](https://github.com/ntkme/github-buttons#readme). Also we were setting `user` and `repo` props, but not using them in the `GithubButtons` component.